packages/@okta/vuepress-site/docs/guides/sign-into-spa/create-okta-application/index.md

```diff
@@ -15,7 +15,9 @@ Start by signing in to the Okta Developer Console:
 
 4. For allowed grant types, make sure that both **Authorization Code** and **Refresh Token** are selected. This enables PKCE flow for your application and the ability to refresh the access token when it expires without prompting the user to reauthenticate. [Refresh token rotation](/docs/guides/refresh-tokens/refresh-token-rotation) is automatically set as the default refresh token behavior.
 
-> **Note:** There is a default grace period for token rotation of 30 seconds. You can [change the value](/docs/guides/refresh-tokens/refresh-token-rotation/#enable-refresh-token-rotation) to any number between 0 and 60 seconds. After the refresh token is rotated, the previous token remains valid for this amount of time to allow clients to get the new token. Using a value of 0 indicates that there is no grace period.
+    Refresh token rotation is an <ApiLifecycle access="ea"/> feature.
```

Comment:
```suggestion
    Refresh token rotation is an <ApiLifecycle access="ea"/> feature. Please email developers@okta.com to enable it for your org.
```

Reply:
I don't think new developers will know what "EA" means and how to get it turned on, hence the added sentence.

Reply:
@mraible - So we send them to developers@okta.com not support to turn on this feature?